DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/TopCourses/Index.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@inject IConfiguration configuration

```

Comment:
We decided on my pull request to leave these as Configuration since this gives Stella a warning instead for some reason.